server/access/rest.go

```diff
@@ -94,7 +95,18 @@ func NewRestServer(logger *zerolog.Logger, adapter *adapters.AccessAdapter, chai
 		}
 	}
 
-	srv, err := rest.NewServer(adapter, fmt.Sprintf("%s:3333", host), debugLogger, chain, restCollector)
+	srv, err := rest.NewServer(
+		adapter,
+		rest.Config{
+			ListenAddress: fmt.Sprintf("%s:3333", host),
+		},
+		debugLogger,
+		chain,
+		restCollector,
+		&state_stream.StateStreamBackend{},
```

Comment:
maybe nil here? Probably we don't have streaming in emulator.

Reply:
You're right, we don't have it yet. I just added it to make it easier for the fellow dev that will add it :sweat_smile: